virt/virt-4-8-release-notes.adoc

```diff
@@ -53,6 +53,11 @@ The SVVP Certification applies to:
 ** Storage
 ** Guest memory swapping
 
+[id="virt-4-8-quick-starts"]
+=== Quick starts
+
+Quick start tours are available for several {VirtProductName} features. To view the tours, click the *?* icon in the masthead of the {VirtProductName} console and select *Quick Starts*.
```

Comment:
Can we add further instructions on how to filter the page to show just the virt quick starts?

Reply:
```suggestion
Quick start tours are available for several {VirtProductName} features. To view the tours, click the *?* icon in the masthead of the {VirtProductName} console and select *Quick Starts*.
You can filter the available quickstarts with the `virtualization` keyword in the filter field.
```

Reply:
Added in new PR: https://github.com/openshift/openshift-docs/pull/34611